### PR TITLE
Proposed section authors (based on authorship form)

### PR DESCRIPTION
**Pull request by adrn**
I have proposed some names for people to contribute text to sections in the paper based on people who replied "willing to contribute text" on the authorship form. This is in no way binding, and you should feel free to use a "suggestion" to remove your name if you do not feel that you can contribute! There are also a number of "unfilled" author roles in sections: Please feel free to sign up to contribute this text! Any unfilled sections will be written by the Paper 3 team (@adrn @pllim @nmearl).

### CODE REVIEW

File: src/ms.tex [hunk 1]
```diff
@@ -184,15 +197,18 @@ \subsection{Highlighted Feature Development} \label{sec:core-features}
 
 \section{Major Updates in the Astropy Project} \label{sec:project-updates}
 
-
 \subsection{Project governance} \label{sec:project-governance}
 
-New CoCo and election overview.
+\secauthor{Erik Tollerud}
```
Comment (adrn) [inline src/ms.tex:202]: @eteq : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

File: src/ms.tex [hunk 2]
```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
 % Transition of needs: From new features, to community development, sustainability, ...
 % - pipeline between User, Participant, Code contributor, Maintainer, Coordinator.
 
+\subsection{Inclusion, Diversity, and Equity Programs} \label{sec:project-ide}
+
+\secauthor{Lía Corrales}
```
Comment (adrn) [inline src/ms.tex:226]: @eblur : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

File: src/ms.tex [hunk 3]
```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
...
 
 
 \subsection{Funding} \label{sec:project-funding}
 
-New funding sources.
-% Funding received:
-% - Moore
-% - NASA
+\secauthor{Aarya Patil}
```
Comment (adrn) [inline src/ms.tex:231]: @aaryapatil : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

File: src/ms.tex [hunk 4]
```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
...
-Infrastructure packages that exist to support and help others maintain their package infrastructure.
+\secunfilled
+
+Infrastructure packages that exist to support and help others maintain their
+package infrastructure.
 
 \subsection{Affiliated packages}
 
-New packages and major updates!
+\secauthor{Matt Craig, Brett Morris}
```
Comment (adrn) [inline src/ms.tex:249]: @mwcraig @bmorris3 : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

File: src/ms.tex [hunk 5]
```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
...
+
+Highlight a few new affiliated packages and major updates to existing ones.
+Include a big table of all affiliated packages and references (as in v2.0
+paper).
+
+\subsection{Connections with data archives}
 
-\subsection{Connections with observatories}
+\secauthor{Adam Ginsburg}
```
Comment (adrn) [inline src/ms.tex:257]: @keflavich : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

File: src/ms.tex [hunk 6]
```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
...
+contributed to Astropy, and vice versa.
+
+JWST
+\secunfilled
 
 Gemini
+\secunfilled
 
+Cherenkov Telescope Array
+\secauthor{Axel Donath, Max Noethe}
```
Comment (adrn) [inline src/ms.tex:273]: @adonath @maxnoe : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

File: src/ms.tex [hunk 7]
```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
...
+
+Rubin Observatory
+\secunfilled
 
 \section{Future Plans for the Astropy Project} \label{sec:future}
 
 \subsection{Roadmap}
 
+\secauthor{Clara Brasseur}
```
Comment (adrn) [inline src/ms.tex:282]: @ceb8 : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

File: src/ms.tex [hunk 8]
```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
...
+
+Overview of roadmap and some highlights.
+
 \section{Learn Astropy} \label{sec:learn}
 
-Current status and scope
+\secauthor{Lía Corrales, David Shupe + Learn team}
```
Comment (adrn) [inline src/ms.tex:288]: @eblur @stargaser : Please us a "Suggestion" to replace your name with `\secunfilled` if you are not able to contribute text to this section.

File: src/ms.tex [hunk 9]
```diff
@@ -205,52 +221,90 @@ \subsection{Contributor base} \label{sec:project-contributors}
...
+\secunfilled
 
 Gemini
+\secunfilled
 
+Cherenkov Telescope Array
+\secauthor{Axel Donath, Max Noethe}
+
+Rubin Observatory
+\secunfilled
```
Comment (lpsinger) [inline src/ms.tex:277]:
```suggestion

LIGO/Virgo/KAGRA
\secauthor{Leo Singer}
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: src/ms.tex (hunk 9 above)
Comment (adrn) [reply src/ms.tex:277]: Thanks @lpsinger -- great idea!

File: src/ms.tex (hunk 6 above)
Comment (maxnoe) [reply src/ms.tex:273]: It's fine from my side

File: src/ms.tex (hunk 8 above)
Comment (stargaser) [reply src/ms.tex:290]: I'm all in!!!

**Comment by pllim** [comment]
Y'all better volunteer because if I write it, I will insert a lot of puns. `#notathreatorisit`

File: src/ms.tex [hunk 10]
```diff
@@ -205,52 +221,92 @@ \subsection{Contributor base} \label{sec:project-contributors}
...
-JWST (motivated by excitement around the launch?)
+Summary of Astroquery and engagement with archives. Cite astroquery paper.
+
+\subsection{Connections with Observatories and Missions}
+
+Brief summary of efforts in observatory- or mission-driven development that have
+contributed to Astropy, and vice versa.
+
+JWST
+\secunfilled
```
Comment (pllim) [inline src/ms.tex:267]: @larrybradley and/or @nden , are you interested in writing this section, or volunteering someone from your branch involved in Astropy/JWST to write it?

File: src/ms.tex [hunk 11]
```diff
@@ -205,52 +221,92 @@ \subsection{Contributor base} \label{sec:project-contributors}
...
+
+\subsection{Connections with data archives}
 
-\subsection{Connections with observatories}
+\secauthor{Adam Ginsburg}
 
-JWST (motivated by excitement around the launch?)
+Summary of Astroquery and engagement with archives. Cite astroquery paper.
+
+\subsection{Connections with Observatories and Missions}
```
Comment (pllim) [inline src/ms.tex:261]: @adrn and/or @nmearl , if we cannot find people to write certain observatory sections, is the next step to remove those sections from the paper altogether?

File: src/ms.tex (hunk 6 above)
Comment (adonath) [reply src/ms.tex:273]: Fine for me too...

File: src/ms.tex [hunk 12]
```diff
@@ -205,52 +221,92 @@ \subsection{Contributor base} \label{sec:project-contributors}
...
+\secauthor{Lía Corrales}
 
 
 \subsection{Funding} \label{sec:project-funding}
 
-New funding sources.
-% Funding received:
-% - Moore
-% - NASA
+\secauthor{Aarya Patil}
```
Comment (sebaguro) [inline src/ms.tex:231]: % Astronomy Open Development built on-top of Open Blockchain Protocol Standards (perhaps subsubsection)

File: src/ms.tex (hunk 10 above)
Comment (larrybradley) [reply src/ms.tex:267]: Yes, I can write the JWST section.

Comment (pllim) [inline src/ms.tex:267]:
```suggestion
\secauthor{Larry Bradley}
```

File: src/ms.tex (hunk 3 above)
Comment (aaryapatil) [reply src/ms.tex:231]: Yes, I can write this section with some help from the finance team 💸

File: src/ms.tex (hunk 8 above)
Comment (eblur) [reply src/ms.tex:290]: Let's discuss and work on this section in the Learn telecons next semester

File: src/ms.tex (hunk 2 above)
Comment (eblur) [reply src/ms.tex:226]: Okay. I would appreciate one other person to help with this section.

File: src/ms.tex (hunk 4 above)
Comment (mwcraig) [reply src/ms.tex:249]: I'm happy to work on this part

File: src/ms.tex (hunk 12 above)
Comment (adrn) [reply src/ms.tex:231]: @sebaguro thanks for the idea here, but as I mentioned in my email, I think this is out of scope for this paper.

File: src/ms.tex (hunk 1 above)
Comment (eteq) [reply src/ms.tex:202]: Happy to write this!

File: src/ms.tex (hunk 7 above)
Comment (ceb8) [reply src/ms.tex:284]: Happy to write this section.

**Review by nmearl — APPROVED** [review]

**Pull request merged**